jparsec-examples/src/test/java/org/codehaus/jparsec/examples/statement/FirstTest.java

```diff
@@ -0,0 +1,97 @@
+package org.codehaus.jparsec.examples.statement;
+
+import static org.junit.Assert.assertEquals;
+
+import org.codehaus.jparsec.Parser;
+import org.codehaus.jparsec.Parsers;
+import org.codehaus.jparsec.Scanners;
+import org.codehaus.jparsec.Terminals;
+import org.codehaus.jparsec.examples.statement.ast.DoubleExpression;
+import org.codehaus.jparsec.examples.statement.ast.SingleExpression;
+import org.junit.Test;
+
+public class FirstTest {
+
+	//low-level grammar
+	private static final Terminals OPERATORS =
+			Terminals.operators("=", "readonly", "var");
+
+	//for integers and identifiers we need a PARSER and a TOKENIZER for each
+	//here are the parsers
+	public static Parser<String> identSyntacticParser = Terminals.Identifier.PARSER;
```

Comment:
Can these static parsers be private static final? Also if they are constants they should be named as FOO_PARSER.